packages/formdescriptioneditors/backend/sirius-components-collaborative-formdescriptioneditors/src/test/java/org/eclipse/sirius/components/collaborative/formdescriptioneditors/handlers/AddGroupEventHandlerTests.java

```diff
@@ -0,0 +1,79 @@
+
+import java.util.Optional;
+import java.util.UUID;
+
+import org.eclipse.sirius.components.collaborative.api.ChangeDescription;
+import org.eclipse.sirius.components.collaborative.api.ChangeKind;
+import org.eclipse.sirius.components.collaborative.formdescriptioneditors.FormDescriptionEditorContext;
+import org.eclipse.sirius.components.collaborative.formdescriptioneditors.TestFormDescriptionEditorBuilder;
+import org.eclipse.sirius.components.collaborative.formdescriptioneditors.api.IFormDescriptionEditorContext;
+import org.eclipse.sirius.components.collaborative.formdescriptioneditors.dto.AddGroupInput;
+import org.eclipse.sirius.components.collaborative.formdescriptioneditors.messages.ICollaborativeFormDescriptionEditorMessageService;
+import org.eclipse.sirius.components.core.api.IEditingContext;
+import org.eclipse.sirius.components.core.api.IObjectService;
+import org.eclipse.sirius.components.core.api.IPayload;
+import org.eclipse.sirius.components.core.api.SuccessPayload;
+import org.eclipse.sirius.components.view.FormDescription;
+import org.eclipse.sirius.components.view.PageDescription;
+import org.eclipse.sirius.components.view.ViewFactory;
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.simple.SimpleMeterRegistry;
+import reactor.core.publisher.Sinks;
+
+/**
+ * Tests of the add group event handler.
+ *
+ * @author frouene
+ */
+public class AddGroupEventHandlerTests {
+
+    @Test
+    public void testAddGroupAction() {
+
+        FormDescription formDescription = ViewFactory.eINSTANCE.createFormDescription();
+        PageDescription pageDescription = ViewFactory.eINSTANCE.createPageDescription();
+        formDescription.getPages().add(pageDescription);
+
+        var objectService = new IObjectService.NoOp() {
+            @Override
+            public Optional<Object> getObject(IEditingContext editingContext, String objectId) {
+                return Optional.of(pageDescription);
+            }
+        };
+        var handler = new AddGroupEventHandler(objectService, new ICollaborativeFormDescriptionEditorMessageService.NoOp(), new SimpleMeterRegistry());
+        var input = new AddGroupInput(UUID.randomUUID(), "editingContextId", "representationId", "pageId", 0);
+
+        assertThat(handler.canHandle(input)).isTrue();
+
+        Sinks.One<IPayload> payloadSink = Sinks.one();
+        Sinks.Many<ChangeDescription> changeDescriptionSink = Sinks.many().unicast().onBackpressureBuffer();
+        IFormDescriptionEditorContext formDescriptionEditorContext = new FormDescriptionEditorContext(new TestFormDescriptionEditorBuilder().getFormDescriptionEditor(UUID.randomUUID()
+                .toString()));
+
+        handler.handle(payloadSink, changeDescriptionSink, new IEditingContext.NoOp(), formDescriptionEditorContext, input);
+
+        ChangeDescription changeDescription = changeDescriptionSink.asFlux().blockFirst();
+        assert changeDescription != null;
```

Comment:
`assertThat(changeDescription).isNotNull()`. We do not use `assert` in our code base, and I don't think asserations are ever enabled (`-ea`).